internal/infrastructure/kubernetes/proxy/resource.go

```diff
@@ -154,6 +159,14 @@ func expectedProxyContainers(infra *ir.ProxyInfra, deploymentConfig *egcfgv1a1.K
 			VolumeMounts:             expectedContainerVolumeMounts(deploymentConfig),
 			TerminationMessagePolicy: corev1.TerminationMessageReadFile,
 			TerminationMessagePath:   "/dev/termination-log",
+			ReadinessProbe: &corev1.Probe{
+				ProbeHandler: corev1.ProbeHandler{
+					HTTPGet: &corev1.HTTPGetAction{
+						Path: envoyReadinessPath,
```

Comment:
looks like `Host` option is also available which can be set to `localhost`

Reply:
IIRC, that only work for hostnetwork?